seo: Add Meta descriptions per page (#761)
FlowFuse/node-red-dashboard: docs/nodes/widgets/ui-control.md

---
{

>     "description": "Master the ui-control widget in Node-RED Dashboard 2.0 to manage dashboard states and data flows.",

Reviewer: The ui-control is more about dynamically updating the elements from node-red side. Not sure this description fits.

Author: ```suggestion
    "description": "Dynamically control elements on your dashboard through ui-control for Node-RED Dashboard 2.0.",
```
